java/node/src/main/java/com/codeheadsystems/dstore/node/dao/TenantTableDao.java

```diff
@@ -213,15 +152,31 @@ public List<String> allTenantTables(final String tenantId) {
    */
   public boolean delete(final String tenantId, final String tableName) {
     LOGGER.trace("delete({})", tenantId);
-    return sqlEngine.executePreparedInternal("delete from NODE_TENANT_TABLES where RID_TENANT = ? and TABLE_NAME = ?", (ps) -> {
-      try {
-        ps.setString(1, tenantId);
-        ps.setString(2, tableName);
-        ps.execute();
-        return ps.getUpdateCount() > 0;
-      } catch (SQLException e) {
-        throw new IllegalArgumentException("Unable to delete a tenant", e);
-      }
-    });
+    return internalJdbi.withHandle(handle -> handle
+        .createUpdate("delete from NODE_TENANT_TABLES where RID_TENANT = :tenant_id and TABLE_NAME = :table_name")
+        .bind("tenant_id", tenantId)
+        .bind("table_name", tableName)
+        .execute() > 0
+    );
+  }
+
+  private static class TenantTableRowMapper implements RowMapper<TenantTable> {
```

Comment:
Once we fix up the ridTenant we may be able to use the bindPojo instead. (Unsure if the identifier makes that just more difficult or impossible...)